src/azul/plugins/metadata/hca/transform.py

```diff
@@ -3,7 +3,6 @@
     abstractmethod,
```

Comment:
PyCharm produces 8 type warnings for this module, with your changes. Zero without.